aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -742,6 +745,41 @@ module supra_framework::pbo_delegation_pool {
         staking_config::get_recurring_lockup_duration(&config) / 4
     }
 
+    /// Initialize a delegation pool without actual coin but withdraw from the owner's account.
+    public fun initialize_delegation_pool_no_coin(
```

Comment:
with_amout

Reply:
Why has the name not been changed yet? Change it to `_with_amount` @axiongsupra

Reply:
Sorry, forget to push my local branch to remote.